Library/Formula/qt.rb

```diff
@@ -25,6 +25,9 @@ class Qt < Formula
     sha256 "c8a0fa819c8012a7cb70e902abb7133fc05235881ce230235d93719c47650c4e"
   end
 
+  # Make `HOMEBREW_PREFIX/lib/qt4/plugins` an additional plugin search path.
+  patch :DATA
```

Comment:
I'd like that to be their decision rather than ours and see if we can't work out a patchless or upstream-acceptable patch solution as we don't want to be carrying patches indefinitely (particularly if upstream doesn't have a change to review the code).